lago/cmd.py

```diff
@@ -277,6 +277,25 @@ def do_stop(prefix, vm_names, **kwargs):
     prefix.stop(vm_names=vm_names)
 
 
+@lago.plugins.cli.cli_plugin(help='Shutdown and destroy lago resources')
+@lago.plugins.cli.cli_plugin_add_argument(
+    '-r',
+    '--reboot',
```

Comment:
No.
Shutdown will keep the vm off, while reboot will turn it off and then power it on.
This is equivalent to the shell command ```shutdown```.